### PR TITLE
Enlarge the SearchFilter's 'X' hitbox

### PR DESCRIPTION
**Pull request by cjb**
@keybase/picnicsquad CC @keybase/react-hackers 

Screen Shot 2019-09-06 at 10 28 58 AM

### CODE REVIEW

File: shared/common-adapters/search-filter.tsx [hunk 1]
```diff
@@ -224,13 +224,14 @@ class SearchFilter extends React.PureComponent<Props, State> {
...
     if (Styles.isMobile) {
       return (
-        <Kb.Icon
-          type="iconfont-remove"
-          sizeType={this._iconSizeType()}
-          onClick={this.props.mobileCancelButton ? this._clear : this._cancel}
-          color={this._iconColor()}
-          style={styles.removeIconNonFullWidth}
-        />
+        <Kb.ClickableBox onClick={this.props.mobileCancelButton ? this._clear : this._cancel}>
```
Comment (chrisnojima) [inline shared/common-adapters/search-filter.tsx]: i believe you can just do this with padding, but this is ok also

Comment (cjb) [reply shared/common-adapters/search-filter.tsx]: Didn't seem to work, but maybe I'm doing something wrong, adding padding to the icon still gives:

Screen Shot 2019-09-06 at 11 53 33 AM

Comment (chrisnojima) [reply shared/common-adapters/search-filter.tsx]: Did add padding to the original or on top of this version. Likely only works with an onclick

Comment (cjb) [reply shared/common-adapters/search-filter.tsx]: to original

**Comment by cjb** [comment]
@chrisnojima Thumb up or down on this one?

**Pull request merged**